svg/elements/altGlyph.json

```diff
@@ -22,12 +22,8 @@
               "version_added": null
             },
             "oculus": "mirror",
-            "opera": {
-              "version_added": "10.6"
-            },
-            "opera_android": {
-              "version_added": "11"
-            },
+            "opera": "mirror",
```

Comment:
Can you also make the other data consistent with the interface? In particular, it was removed from Chrome.